linera-execution/src/lib.rs

```diff
@@ -50,6 +51,136 @@ use serde::{Deserialize, Serialize};
 use std::{io, path::Path, str::FromStr, sync::Arc};
 use thiserror::Error;
 
+/// Substracts an amount from a balance and reports an error if that is impossible
+pub fn sub_assign_fees(balance: &mut Amount, fees: Amount) -> Result<(), PricingError> {
```

Comment:
ok so for a public APIs in general I try to use methods because explicitly importing each public function is tedious.

Reply:
So, far this is a single import. We can think of something else in the future if that grows out.